File: src/app/services/tabela.cultivo.service.ts
import {BaseCrudService} from "../core/base.crud.service";
import {TabelaCultivo} from "../model/tabela.cultivo";
import {Injectable} from "@angular/core";
import {HttpClient} from "@angular/common/http";

@Injectable({
  providedIn: "root"
})
export class TabelaCultivoService extends BaseCrudService<TabelaCultivo> {

  constructor(public http: HttpClient) {
    super(http, "/tabela-cultivo");
  }
}

File: src/main/java/br/com/getservicos/fishControl/service/api/TabelaCultivoService.java
package br.com.getservicos.fishControl.service.api;

import br.com.getservicos.fishControl.core.service.api.BaseCrudService;
import br.com.getservicos.fishControl.model.TabelaCultivo;

public interface TabelaCultivoService extends BaseCrudService<TabelaCultivo, Integer> {
    TabelaCultivo getByTanque(String tanque);
}
